openmp/libomptarget/plugins-nextgen/common/PluginInterface/PluginInterface.cpp

```diff
@@ -49,40 +49,88 @@ struct RecordReplayTy {
   void *MemoryStart;
   void *MemoryPtr;
   size_t MemorySize;
```

Comment:
Not about this patch, but we really need to add documentation here, MemorySize doesn't really imply what this variable tracks. Probably renaming and docs is adequate.